modules/rhel-compute-requirements.adoc

```diff
@@ -23,7 +23,11 @@ ifdef::openshift-enterprise,openshift-webscale[]
 +
 [IMPORTANT]
 ====
-Only RHEL 7.9 is supported in {product-title} {product-version}. You must not upgrade your compute machines to RHEL 8.
+Adding RHEL 7 compute machines to an {product-title} cluster is deprecated. Deprecated functionality is still included in {product-title} and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments.
```

Comment:
I see none of this module uses the RHEL attributes (`{op-system-base-full}` and `{op-system-base}`). Not sure if it's worth updating now or later, but thought I'd mention it.

Reply:
Good point and worth the update. Resolved. Thanks, @codyhoag!